chapters/ch06.md

```diff
@@ -1,48 +1,48 @@
 
-Dựa trên vị trí người dùng ứng dụng di động của bạn, tập dữ liệu ảnh mèo được tải lên từ bốn khu vực chính: (i) Hoa Kỳ, (ii) Trung Quốc, (iii) Ấn Độ, và (iv) Khu vực khác. Để tạo tập phát triển và tập kiểm tra, giả định như chúng ta lấy dữ liệu từ Hoa Kỳ và Ấn Độ để tạo tập phát triển; Trung Quốc và Khu vực khác để tạo tập kiểm tra. Nói cách khác, chúng ta có thể chọn ngẫu nhiên dữ liệu ảnh từ hai trong bốn khu vực trên làm tập phát triển, dữ liệu ảnh từ hai khu vực còn lại làm tập kiểm tra. Phân chia vậy liệu có đúng hay không?
+Ứng dụng của bạn được phân khúc thành bốn vùng theo quy mô thị trường: (i) Hoa Kỳ, (ii) Trung Quốc, (iii) Ấn Độ, và (iv) các nước Khác. Để tạo tập phát triển và tập kiểm tra, giả định là chúng ta lấy dữ liệu từ Hoa Kỳ và Ấn Độ để tạo tập phát triển; Trung Quốc và Khác để tạo tập kiểm tra. Nói cách khác, chúng ta có thể chọn ngẫu nhiên dữ liệu ảnh từ hai trong bốn khu vực trên làm tập phát triển, và hai khu vực còn lại làm tập kiểm tra. Đúng không nào?
 
 > Once you define the dev and test sets, your team will be focused on improving dev set performance. Thus, the dev set should reflect the task you want to improve on the most: To do well on all four geographies, and not only two.
 
-Một khi bạn đã phân loại tập phát triển và tập kiểm tra, nhóm của bạn sẽ tập trung cải thiện kết quả trên tập phát triển. Bởi vậy, tập phát triển nên đại diện nhiệm vụ bạn muốn cải thiện nhiều nhất: hoạt động tốt trên dữ liệu không chỉ hai mà cả bốn thị trường.
+Một khi bạn đã xác định được tập phát triển và tập kiểm tra, đội nhóm của bạn sẽ tập trung cải thiện kết quả trên tập phát triển. Do đó tập phát triển nên phản ánh được việc mà bạn muốn cải thiện nhiều nhất: hoạt động tốt trên dữ liệu từ không chỉ hai mà cả bốn khu vực địa lý.
 
 > There is a second problem with having different dev and test set distributions: There is a chance that your team will build something that works well on the dev set, only to find that it does poorly on the test set. I’ve seen this result in much frustration and wasted effort. Avoid letting this happen to you.
 
-Vấn đề thứ hai với việc tập phát triển và tập kiểm tra có phân phối khác nhau: Có khả năng nhóm của bạn xây dựng được một mô hình hoạt động tốt trên tập phát triển nhưng lại kém trên tập kiểm tra. Tôi đã từng thấy việc này (việc có phân phối khác nhau của hai tập dữ liệu) dẫn đến những kết quả đáng thất vọng và sự lãng phí về công sức. Hãy tránh để điều này xảy ra.
+Vấn đề thứ hai đối với sự phân phối của tập phát triển và tập kiểm tra: Có khả năng đội nhóm của bạn xây dựng được một thứ gì đó hoạt động tốt trên tập phát triển nhưng lại kém trên tập kiểm tra. Tôi đã từng thấy điều này  dẫn đến những kết quả đáng thất vọng và lãng phí công sức ra sao . Hãy tránh để nó xảy ra với bạn.
```

Comment:
Chỗ này bạn bỏ hẳn cụm "có phân phối khác nhau" trong khi bản gốc có "diferent ... distributions". Việc này có ý nghĩa rất quan trọng, mình không hiểu sao bạn lại bỏ đi.

Reply:
* Bạn dịch "thứ gì đó" là word-by-word rồi. Từ "mô hình" ở đây làm cho câu có nghĩa hơn rất nhiều. "Thứ gì đó" rất ít được sử dụng trong văn viết.
* Từ đầu cuốn sách từ "team" đều được dịch là "nhóm" chứ không phải "đội", và càng không phải là "đội nhóm". Tiếng Việt hay nói "làm việc nhóm" chứ không nói "làm việc đội".

Reply:
ok, mình sót chữ "different", sẽ bổ sung

Reply:
Không phải mình dịch word-by-word, mà cố gắng giữ phong cách của tác giả. Nếu "thứ gì đó" nghe không giống văn viết thì sửa lại thành "điều gì đó". Ở đây tác giả không nói đó là cái gì, nên mình nghĩ dịch là "mô hình" thì không chính xác

Reply:
"something that works well on the dev set" --> chỉ có "mô hình", "bộ phân loại" thì mới "hoạt động tốt trên tập phát triển" thôi bạn. Có thể như bạn nói, tác giả không nói đó là cái gì, nhưng chẳng phải phần lớn chúng ta khi nhắc đến machine learning đều nghĩ đến build model?

Reply:
> 
> 
> "something that works well on the dev set" --> chỉ có "mô hình", "bộ phân loại" thì mới "hoạt động tốt trên tập phát triển" thôi bạn. Có thể như bạn nói, tác giả không nói đó là cái gì, nhưng chẳng phải phần lớn chúng ta khi nhắc đến machine learning đều nghĩ đến build model?

Mình không cho rằng chỉ có mô hình, bởi ngay phía dưới tác giả đã đưa ra ví dụ về "your team develops a system that works well on the dev set but not the test set"